pkgs/applications/blockchains/groestlcoin/default.nix

```
@@ -0,0 +1,101 @@
+{ lib
+, stdenv
+, fetchurl
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, util-linux
+, hexdump
+, autoSignDarwinBinariesHook
+, wrapQtAppsHook ? null
+, boost
+, libevent
+, miniupnpc
+, zeromq
+, zlib
+, db53
+, sqlite
+, qrencode
+, qtbase ? null
+, qttools ? null
+, python3
+, withGui
+, withWallet ? true
+}:
+
+with lib;
```

```suggestion
```

Please use with lib only over small scopes like meta.

---

if I remove this the all sorts of error messages starts to appear (for meta maintainer) but also non meta like optional.

---

For now I will leave this in so it actually builds

---

Maybe I could fix those with: 

```suggestion
meta = with lib; {
```

and 

```suggestion
configureFlags = with lib; [
```

But not sure if other errors will arrise

---

That is not true. It is used on a lot but not all and it shouldn't be used over the entire package definition because it costs performance.

---

removed now